plugins/main/public/styles/index.ts

```diff
@@ -27,7 +27,5 @@ const IS_DARK_THEME = getUiSettings().get('theme:darkMode');
 /* tslint-disable no-undef */
 if (IS_DARK_THEME) {
   import('./theme/dark/index.dark.scss').then();
-}else{
-  import('./theme/light/index.light.scss').then();
```

Comment:
**thought**: why is this removed?

Reply:
The only thing it contained was the `Home -> Overview` icon colors, that's why I removed it.